setup.py

```diff
@@ -158,6 +160,11 @@ def get_data_files(name, version, fullname):
         set_conf_files(data_files, src=["config/debian/waagent.conf"])
         set_logrotate_files(data_files)
         set_udev_files(data_files, dest="/lib/udev/rules.d")
+        # All Debian versions since Jessie use systemd
```

Comment:
Does this means we will update for each new version of debian? Can't you just specify the old supported version and say the other ones use systemd ?

Reply:
Some additional information on platforms, just tucking it away somewhere so I can use it later.  Apparently it isn't so easy to identify debian releases with platform.dist() - of course it's been deprecated since python 2.6, but I have addressed that in another PR.
```
debian:jessie
    >>> print(platform.dist())
    ('debian', '8.11', '')

debian:stretch
    >>> print(platform.dist())
    ('debian', '9.7', '')

debian:buster
    >>> print(platform.dist())
    ('debian', 'buster/sid', '')
```

Reply:
@plessisa please review the change, I removed the named comparison

Reply:
LGTM